bootloader_samples/smp/mcuboot_smp_uart/prj.conf

```diff
@@ -1,4 +1,21 @@
 CONFIG_BOOTLOADER_MCUBOOT=y
 CONFIG_MCUMGR=y
-CONFIG_MCUMGR_SMP_UART=y
-CONFIG_MCUMGR_CMD_IMG_MGMT=y
+CONFIG_MCUMGR_TRANSPORT_UART=y
+CONFIG_MCUMGR_GRP_IMG=y
+
+# CONFIG_MCUMGR_TRANSPORT_UART dependencies
+CONFIG_BASE64=y
+
+# CONFIG_MCUMGR_GRP_IMG dependencies
```

Comment:
These used to be selected by CONFIG_MCUMGR_CMD_IMG_MGMT.
https://github.com/nrfconnect/sdk-zephyr/blob/v3.2.99-ncs2/subsys/mgmt/mcumgr/grp/img_mgmt/Kconfig L5-L12
Now they are not selected by, only depended on by CONFIG_MCUMGR_GRP_IMG
https://github.com/nrfconnect/sdk-zephyr/blob/v3.3.99-ncs1/subsys/mgmt/mcumgr/grp/img_mgmt/Kconfig L15-L23